JSDemos/Demos/TreeView/HierarchicalDataStructure/description.md

```diff
@@ -1 +1,5 @@
-The TreeView component displays a&nbsp;hierarchical collection of&nbsp;items using a&nbsp;traditional tree structure. TreeView nodes can be&nbsp;expanded and collapsed to&nbsp;display sub-items.
\ No newline at end of file
+To bind the TreeView to a hierarchical array, assign it to the [items[]](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/items/) property. If you need to use a [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) object and its diverse API, specify the [dataSource](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#dataSource) property instead.
+
+No matter which property is used, each hierarchical node should include a `text`, a unique `id`, optional developer-defined fields, and an `items` array if a node nests others. If the **items[]** TreeView property is specified, fields from its API can also be included.
```

Comment:
This sentence is definitely too long. Let's split it: 
Regardless of the property used, each hierarchical node should include `text` and unique `id` fields. Specify the `items[]` array for the nodes that nest other nodes.

>If the **items[]** TreeView property is specified, fields from its API can also be included.

We can easily get rid of passive here:
You can also specify fields from the TreeView.**items[]** API if you use this property.